### PR TITLE
chore(release): Release v.0.2.0

### PR DESCRIPTION
**Pull request by jophish**
standard-version set this to v0.2.0 instead of v0.1.0; not too concerned since we're still on major version 0.

### CODE REVIEW

File: test/index.test.ts [hunk 1]
```diff
@@ -206,7 +206,10 @@ describe('FiatConnect SDK', () => {
         'https://fiat-connect-api.com/auth/login',
         expect.objectContaining({
           method: 'POST',
-          headers: { 'Content-Type': 'application/json', Authorization: 'Bearer api-key' },
+          headers: {
```
Comment (jophish) [inline test/index.test.ts:209]: linter did this :eyes:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+* **auth:** implement siwe ([#10](https://github.com/fiatconnect/fiatconnect-sdk/issues/10)) ([6b8f1d3](https://github.com/fiatconnect/fiatconnect-sdk/commit/6b8f1d385b3e099ba9124f5446a7fc8b6dd1db7d))
+* **auth:** update login to always create a session ([#12](https://github.com/fiatconnect/fiatconnect-sdk/issues/12)) ([8f552d7](https://github.com/fiatconnect/fiatconnect-sdk/commit/8f552d7f668d967989e57f35d9e8a0a7ac00f060))
+* **clock:** Add clock support to SDK ([#11](https://github.com/fiatconnect/fiatconnect-sdk/issues/11)) ([3ad99a1](https://github.com/fiatconnect/fiatconnect-sdk/commit/3ad99a1c177df3dd49ca2aa1ff2bef9d48dcf48d))
+
+
+* added provider name and icon url to config ([#7](https://github.com/fiatconnect/fiatconnect-sdk/issues/7)) ([ec3e1f2](https://github.com/fiatconnect/fiatconnect-sdk/commit/ec3e1f2cc72e4fa8ea654cf2f4c603fba5b62a77))
+* updated types ([#8](https://github.com/fiatconnect/fiatconnect-sdk/issues/8)) ([4593436](https://github.com/fiatconnect/fiatconnect-sdk/commit/4593436cfa3437cc089d02f509ac6faa0cf60469))
+
+# Changelog
```
Comment (cajubelt) [inline CHANGELOG.md:33]: this section is now present twice in the file

File: test/index.test.ts (hunk 1 above)
Comment (cajubelt) [reply test/index.test.ts:209]: nit: ideally we'd do this in separate PRs

File: CHANGELOG.md (hunk 2 above)
Comment (jophish) [reply CHANGELOG.md]: ty!

File: test/index.test.ts (hunk 1 above)
Comment (jophish) [reply test/index.test.ts:209]: https://github.com/fiatconnect/fiatconnect-sdk/pull/16

**Review by silasbw — APPROVED** [review]
I assume this is all generated by `yarn release`

**Pull request merged**